src/main/webapp/CreateProfilePage.html

```diff
@@ -9,14 +9,20 @@
   <body>
     <div class="container">
       <h1 class="text-center">Create Your Profile Page</h1>
-      <form action="/profile-data" method="POST" novalidate>
+      <form
+        action="/profile-data"
+        method="POST"
+        class="needs-validation"
+        novalidate
+      >
         <div class="form-group col-md-5" style="margin: 0 auto;">
           <label for="name">Name:</label><br />
           <input
             type="text"
             class="form-control"
             name="name"
             maxlength="160"
+            pattern= "^[a-zA-Z0-9_]+( [a-zA-Z0-9_]+)*" 
```

Comment:
Nice regex pattern! Other characters to consider allowing are a hyphen `-` and a period `.` (e.g. Marc-Antoine, Dr. Smith, J.R.R. Tolkien)

Maybe something like:
`^[\.a-zA-Z0-9_-]+( [\.a-zA-Z0-9_-]+)*`